src/applications/personalization/dashboard-2/tests/unit/healthcare-spinner.unit.spec.jsx

```diff
@@ -1,17 +1,12 @@
 import React from 'react';
 import { expect } from 'chai';
 import { renderInReduxProvider } from '~/platform/testing/unit/react-testing-library-helpers';
-import { resetFetch } from '~/platform/testing/unit/helpers';
 import reducers from '~/applications/personalization/dashboard/reducers';
 import HealthCare from '~/applications/personalization/dashboard-2/components/health-care/HealthCare';
 
 describe('HealthCare component', () => {
   let view;
   let initialState;
-  // If a test mocks fetch and fails to properly restore fetch, and that
-  // offending test runs shortly before this test, it can result in this test to
-  // fail. So we are calling `resetFetch()` defensively.
-  before(resetFetch);
```

Comment:
There's no danger in doing this, right, it's just not necessary any more?

Reply:
Exactly.

Reply:
I did want to pull those calls out, though, because there were a lot of places where the test files had their own resetFetch commands and I really would like to have our fetch implementation more standardized across the repo.